src/lib/y2storage/mountable.rb

```diff
@@ -88,6 +88,14 @@ def mountpoint=(path)
 
     alias_method :mount_point=, :mountpoint=
 
+    # Is the mount persistent?
+    #
+    # @return [Boolean] true if the mount point is saved to /etc/fstab
+    #   (and will be mounted at boot again), false otherwise
```

Comment:
NP: I would add a "TODO" to say that this should be adapted once `MountPoint` wrapper class is there.